UOP1_Project/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,7 +5,10 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
+  - enabled: 0
+    path: 
+    guid: 00000000000000000000000000000000
   - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
-    guid: d1c3109bdb54ad54c8a2b2838528e640
+    path: Assets/Scenes/TitleMenu.unity
```

Comment:
You shouldn't submit this file. Please remove from your PR.

Reply:
Hi Cant we add new Scenes? What is the reason for the request?